### ISSUE TITLE
Add storage capacity check to rig doctor

### ISSUE THREAD

**Issue by grayside** [phase2/rig#44]
Individuals using continuous docker machines that do not perform cleanup will start to hit capacity limits, and this can often take some troubleshooting time to figure out.

Let's add rig doctor checks to warn at something like 85% of /dev/sda1 and /Users capacity.

Maybe take it to an error on 95%?

**Comment by grayside** [comment]
For the main data volume:
```
df -h 2> /dev/null | grep /dev/sda1 | head -1 | awk '{print $5}' | sed 's/%//'
```

For the /Users directory:
```
df -h 2> /dev/null | grep /Users | head -1 | awk '{print $5}' | sed 's/%//'
```

**Comment by febbraro** [comment]
Closed via commit: 91dc8a7f977329b00ca14a2f2b6830ebe2ba1e83
